packages/lesmis/src/api/queries/useMapOverlayReportData.js

```diff
@@ -142,11 +143,6 @@ export const useMapOverlayReportData = ({ entityCode, year }) => {
     setHiddenValues({});
   }, [setHiddenValues, selectedOverlay, entityCode]);
 
-  // reset selected overlay when changing entity
-  useEffect(() => {
-    setSelectedOverlay(null);
-  }, [setSelectedOverlay, entityCode]);
```

Comment:
Is removing this fine because when changing entity it changes the url to that entity's base url (no search) anyway?

Reply:
yes exactly!